Realm/RLMRealmConfiguration.h

```diff
@@ -60,6 +60,9 @@ RLM_ASSUME_NONNULL_BEGIN
 /// The block which migrates the Realm to the current version.
 @property (nonatomic, copy, nullable) RLMMigrationBlock migrationBlock;
 
+/// A custom schema to use for the Realm.
+@property (nonatomic, copy, nullable) RLMSchema *customSchema;
```

Comment:
TA wants us to keep the word "schema" (and the general concept) out of our public API as much as possible, so this should probably be `NSArray<Class> *objectClasses;`. Setter should probably construct the RLMSchema object and store it in a private property as opposed to creating it when the Realm is opened to avoid having creating a bunch of Realm instances from a single config be slow.